editoast/Cargo.toml

```diff
@@ -16,7 +16,6 @@ members = [
 ]
 
 [workspace.lints.rust]
-dependency_on_unit_never_type_fallback = "allow" # this is caused by tracing, we can't do anything about it
```

Comment:
This warning was disabled because it occurs on nightly and doesn't seem to be fixable on our end. Did it disappear on recent builds?

Reply:
Yes, I got a warning on this lint not being recognized for rust 1.80.1
```
warning[E0602]: unknown lint: `dependency_on_unit_never_type_fallback`
```

Reply:
Yep, it generates a warning on `stable`. Resolving since it's our target, not nightly.